features/fixtures/laravel10/Dockerfile

```diff
@@ -0,0 +1,23 @@
+ARG PHP_VERSION
+FROM php:$PHP_VERSION
+
+RUN apt-get update && \
+  apt-get install -y --no-install-recommends \
+  git \
+  unzip \
+  wget \
+  zip
+
+WORKDIR /app
+
+COPY . .
+COPY --from=composer:2.2 /usr/bin/composer /usr/local/bin/composer
+
+RUN cp .env.example .env
+RUN composer install --no-dev
```

Comment:
Some dev dependencies were causing the "small OOM" test to fail semi-reliably on PHP 8.2 — it failed every time on CI but sometimes passed locally

Given bugsnag is mostly relevant in production, it seems fine to me to avoid installing dev dependencies in these fixtures so that they can't cause flakiness (I'll open a PR for the other fixtures separately)